internal/fs/inode/type_cache.go

```diff
@@ -50,58 +87,126 @@ type typeCache struct {
 	//
 	// INVARIANT: entries.CheckInvariants() does not panic
```

Comment:
we have removed the first invariant check, right ?

Reply:
Right. that line of comment was obsolete now. Removed.